### PR TITLE
[11.0][FIX] account_payment_partner: onchange if more than one bank

### PR DESCRIPTION
**Pull request by astirpe**
Fixes https://github.com/OCA/bank-payment/issues/441

### CODE REVIEW

File: account_payment_partner/models/account_invoice.py [hunk 1]
```diff
@@ -27,18 +27,19 @@ def _onchange_partner_id(self):
...
                     self.commercial_partner_id.bank_ids
                 ):
-                    self.partner_bank_id = \
+                    partner_banks = \
                         self.commercial_partner_id.bank_ids.filtered(
                             lambda b: b.company_id == self.company_id or not
                             b.company_id)
-                else:
-                    self.partner_bank_id = False
+                    if partner_banks:
```
Comment (pedrobaeza) [inline account_payment_partner/models/account_invoice.py]: No need to make `if`. If you put at the end `[:1]`, it servers for 0 or more elements to filter the first one.

**Comment by astirpe** [comment]
@pedrobaeza this Runbot warning:

```
2018-01-22 12:58:20,470 168 WARNING openerp_test odoo.addons.sms.wizard.send_sms: The `phonenumbers` Python module is not installed. Try: pip install phonenumbers.
```

is the one (I guess) that makes the runbot icon red. I see this warning in every repository for every new PRs on branch 11.0. Can we install the `phonenumbers`? I don't know if you can do it or is someone else.

**Review by ThijsvOers — APPROVED** [review]
I have tested the latest commit (more than one bank number per supplier in a multi company environment gave an error when creating a vendor bill) and everything is working fine. 

An extra feature request would be to add an extra priority field to the bank number on a vendor/customer. But maybe there is already such a request.

File: account_payment_partner/models/account_invoice.py [hunk 2]
```diff
@@ -27,6 +27,7 @@ def _onchange_partner_id(self):
                     force_company=self.company_id.id
                 ).partner_id.supplier_payment_mode_id
                 self.payment_mode_id = pay_mode
+                self.partner_bank_id = False
```
Comment (pedrobaeza) [inline account_payment_partner/models/account_invoice.py]: This is not needed now

File: account_payment_partner/models/account_invoice.py [hunk 3]
```diff
@@ -36,9 +37,7 @@ def _onchange_partner_id(self):
                     self.partner_bank_id = \
                         self.commercial_partner_id.bank_ids.filtered(
                             lambda b: b.company_id == self.company_id or not
-                            b.company_id)
-                else:
-                    self.partner_bank_id = False
```
Comment (pedrobaeza) [inline account_payment_partner/models/account_invoice.py]: Please keep this for avoiding a double assignation plus possible triggers

**Comment by pedrobaeza** [comment]
The warning is because a module is not correctly informing about a missing library. It should use `logger.info`, not `logger.warn`. Any way, it's easy to fix: add the library to the requirements.txt file in the root of the repository that hosts the module for being automatically installed on runbot.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**